packages/block-editor/src/components/rich-text/index.js

```diff
@@ -592,6 +632,20 @@ export class RichText extends Component {
 		event.preventDefault();
 	}
 
+	/**
+	 * Updates the selection state as soon as possible, event before the next
```

Comment:
It's unclear to me whether the correct reading of this should be:

```suggestion
	 * Updates the selection state as soon as possible, even before the next
```

Or:

```suggestion
	 * Updates the selection state at the soonest possible event before the next
```